dot/network/notifications.go

```diff
@@ -258,7 +263,7 @@ func (s *Service) broadcastExcluding(info *notificationsProtocol, excluding peer
 
 			// we've already completed the handshake with the peer, send message directly
 			logger.Trace("sending message", "protocol", info.protocolID, "peer", peer, "message", msg)
-			err = s.host.send(peer, info.protocolID, msg)
+			err = s.host.send(peer, info.protocolID, msg, *stream)
```

Comment:
same here, remove `stream` from `send` function

Reply:
in general this function needs to check `notificationsProtocol.handshakeData[peer]` for the stream if it already exists for this peer, then use `writeToStream` if the stream is opened and established, otherwise it should use `send` to open a new stream and send the handshake (which is what was previously happening)